test/integration/import_test.go

```diff
@@ -9,14 +9,21 @@ import (
 	errorConstants "github.com/checkmarx/ast-cli/internal/constants/errors"
 	featureFlagsConstants "github.com/checkmarx/ast-cli/internal/constants/feature-flags"
 	"github.com/checkmarx/ast-cli/internal/params"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
 	"github.com/checkmarx/ast-cli/internal/wrappers"
+	"github.com/spf13/viper"
 	"gotest.tools/assert"
 )
 
 func TestImport_ImportSarifFileWithCorrectFlags_CreateImportSuccessfully(t *testing.T) {
 	// createASTIntegrationTestCommand is called just to load the FF values
 	createASTIntegrationTestCommand(t)
-	if !wrappers.FeatureFlags[featureFlagsConstants.ByorEnabled] {
+
```

Comment:
suggestion: extract lines 22-26 to a new func 'isFFByorEnabled'

Reply:
Done